engine/setup/src/include/general.cpp

```diff
@@ -149,7 +172,8 @@ char *replace(char *line, char *search, char *replace, int LENGTH) {
 	return line;
 }
 
-char *strmov(char *to, char *from) {
+char *strmov(char *to, char *from) 
```

Comment:
this should take a boundary parameter 9 `strmov(char*, char*, size_t)` same format as `strnlen/strncat/strncpy`...probably should call it `strnmov` too for consistency